test/parallel/test-fs-access.js

```diff
@@ -94,14 +94,6 @@ assert.throws(function() {
   fs.access(100, fs.F_OK, function(err) {});
 }, /path must be a string or Buffer/);
 
-assert.throws(function() {
-  fs.access(__filename, fs.F_OK);
-}, /"callback" argument must be a function/);
-
-assert.throws(function() {
-  fs.access(__filename, fs.F_OK, {});
-}, /"callback" argument must be a function/);
```

Comment:
I don't think this behavior should change.  Calling `fs.access()` without a callback arguably never makes sense.